values.yaml

```diff
@@ -219,6 +223,16 @@ keda:
       <<: *platformAffinity
     tolerations:
       *platformTolerations
+
+nats:
+  resources:
+    requests:
+      cpu: "5m"
+      memory: "7Mi"
+    limits:
+      cpu: "15m"
+      memory: "20Mi"
```

Comment:
Do we want some larger values here?

Also, should we add NATS to the platform node pools, like we did with KEDA right above this?

Reply:
Those are pretty small, likely for basic testing. Let's turn those up a bit high on purpose than dial them back after we do some testing that puts a close to real load on it. maybe like 250 or 500 for cpu limit and 100megs of memory? We'll need to review its usage after its running in a dev cluster and adjust as needed.

Reply:
done.